package/boot/arm-trusted-firmware-rockchip/Makefile

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (C) 2020 Tobias Maedel <openwrt@tbspace.de>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=arm-trusted-firmware-rockchip
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=https://github.com/ARM-software/arm-trusted-firmware
+PKG_SOURCE_DATE:=2020-01-03
```

Comment:
Why are we not using releases here which are much easier to track?

Reply:
I need commit https://github.com/ARM-software/arm-trusted-firmware/commit/b4899041e5f0b8e8b388c6511b5233516b8785ec which isn't part of any stable release yet. 
I could use 2.3.0-rc0 if that would be preferrable?

Reply:
Yes, that's a much better choice

Reply:
Done.